kpatch/kpatch

```diff
@@ -163,7 +163,16 @@ load_module () {
 	fi
 
 	echo "loading patch module: $1"
-	insmod "$1" "$2"
+	for i in `seq 5`; do
+		if [[ $i != 1 ]]; then
+			echo "retrying..."
+			sleep 2
+		fi
+		insmod "$1" "$2" && break
+	done
+	[[ $i = 5 ]] && die "failed to load module $modname"
```

Comment:
Alternatively move the conditional check at the bottom of the loop.  Any successful ```insmod``` will break the loop and skip the check...  Or do something like: ```insmod "$1" "$2" && loaded=1 && break``` to employ a success flag without having duplicate "5" constants laying about.